nginx.py

```diff
@@ -1,6 +1,28 @@
 from fabric.api import *
+import fabric.contrib.files
+import puppet
+
+maintenance_config = '/etc/nginx/includes/maintenance.conf'
```

Comment:
I was following what we do in https://github.com/alphagov/fabric-scripts/blob/master/locksmith.py#L7